core/src/main/resources/software/amazon/awssdk/core/util/mime.types

```diff
@@ -353,7 +353,7 @@ application/x-futuresplash	spl
 application/x-gtar		gtar
 application/x-gzip		gz
 application/x-hdf		hdf
-application/x-javascript	js
+application/javascript	js
```

Comment:
Is it worth filtering out the ones that don't have extensions?

Reply:
Yes, I got the latest mime.types from apache server and removed the ones without extensions